packages/terra-list/src/ListItem.jsx

```diff
@@ -102,10 +115,25 @@ const ListItem = ({
     attrSpread.onBlur = ListUtils.wrappedEventCallback(onBlur, event => event.currentTarget.setAttribute('data-item-show-focus', 'true'));
     attrSpread.onMouseDown = ListUtils.wrappedEventCallback(onMouseDown, event => event.currentTarget.setAttribute('data-item-show-focus', 'false'));
   }
+  if (isDraggable) {
+    attrSpread['aria-describedby'] = responseId;
+  }
+  const addingIconclassName = cx([
+    'item-fill',
+    (isDraggable) && 'icon-knurling',
+  ]);
 
   return (
     <li {...customProps} {...attrSpread} className={listItemClassNames} ref={refCallback}>
-      <div className={cx('item-fill')} key="item-fill">{children}</div>
+      <div className={addingIconclassName} key="item-fill">
```

Comment:
key should always be unique and should not be set to same string.

Reply:
Changed done. Thanks